### ISSUE TITLE
Course information should be in properly formatted HTML

### ISSUE THREAD

**Issue by jkeck** [sul-dlss/SearchWorks#428]
The information for course reserves is done in a different way from all of our other data.  Course reserves data uses muted spans to try and mimic a label that we could get from a proper `dt`/`dd`. 
## ![crez-ui](https://cloud.githubusercontent.com/assets/96776/3531157/86145b48-07ac-11e4-9de5-2f3ff7fd6fee.png)

![crez-markup](https://cloud.githubusercontent.com/assets/96776/3531161/8a415c52-07ac-11e4-92f7-08a3a665d9cf.png)
